tabpy-server/tabpy_server/handlers/status_handler.py

```diff
@@ -17,8 +14,6 @@ def get(self):
 
         self._add_CORS_header()
 
-        logger.debug(self.append_request_context(
```

Comment:
Why remove this?

Reply:
base class automatically logs request context which includes method and URL